editor/editor_node.cpp

```diff
@@ -3933,7 +3932,7 @@ void EditorNode::register_editor_types() {
 
 	// FIXME: Is this stuff obsolete, or should it be ported to new APIs?
 	GDREGISTER_CLASS(EditorScenePostImport);
-	//ClassDB::register_type<EditorImportExport>();
+	// ClassDB::register_type<EditorImportExport>();
```

Comment:
Seems like this type no longer exists.

Reply:
If we remove the dead code I guess we can also remove the `FIXME` comment above. I added this more than a year ago, if this didn't change since it's likely not relevant anymore.

Reply:
Isn't it for EditorScenePostImport? Or was EditorScenePostImport just added between the comment and the commented out code?

Reply:
Ok, removed both comments.

Reply:
In PR #10656
```
//ClassDB::register_type<EditorImporter>();
//ClassDB::register_type<EditorPostImport>();
```
was replaced with:
```
// FIXME: Is this stuff obsolete, or should it be ported to new APIs?
//ClassDB::register_class<EditorExportPlugin>();
//ClassDB::register_class<EditorScenePostImport>();
//ClassDB::register_type<EditorImportExport>();
```

Reply:
IIRC EditorScenePostImport was ported recently. The only thing left would be EditorImportExport, but it's not clear what it's supposed to do. Also it's 5 years old...

Reply:
I did a quick research with git log, but couldn't find the exact commit where EditorImportExport was removed or renamed. But in the end it should be fine to remove this commented-out line.

Reply:
Good to remove.